test/zdtm/static/child_subreaper.c

```diff
@@ -0,0 +1,36 @@
+#include <stdlib.h>
```

Comment:
And it looks like you accidentally merged zdtm test patch with criu patch. Please make them separate patches.